elementary/monitor/cli.py

```diff
@@ -280,6 +287,7 @@ def send_report(
         update_dbt_package,
         slack_token,
         slack_channel_name,
+		  slack_timezone,
```

Comment:
Its a bit confusing to pass a timezone to the send_report command when it doesn't affect anything.

As long as it has no use, let's not add it to this command.